executor.go

```diff
@@ -266,7 +266,9 @@ func (e *Executor) runPass() {
 
 					started := time.Now()
 
-					err = task.Run(ctx, e.ShellRun)
+					if task.Run != nil {
+						err = task.Run(ctx, e.ShellRun)
```

Comment:
should we output something differently for running a group? Also feels like groups shouldn't report their own time, since they will always take 0s. I really hated that behavior from gulp.

Reply:
I made it so that null times aren't reported, and durations are only incremented for non-null runs